### ISSUE TITLE
Publish app not rendering images.

### ISSUE THREAD

**Issue by ronak-k-p** [dotnet/maui#10526]
### Description

Published app doesn't render any images.

### Steps to Reproduce

1. Create new MAUI Project 
    Visual studio creates one templated project with "Hello world text" + one image(dotnet_bot.svg) in center of window.
2.  Publish app using command
     open cmd > set path to project directory (cd <project-directory-path>)
    Run following command
    **`dotnet publish -f net7.0-windows10.0.19041.0 -c Release -p:WindowsPackageType=None  -p:SelfContained=true -p:WindowsAppSDKSelfContained=true`** 
3.  Open <project-directory-path>/bin/release/publish/YourProjectName.exe
    Observe image(dotnet_bot.svg) is not rendering in UI anywhere.

### Link to public reproduction project repository

NA

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows

**Comment by ArrowCase** [comment]
The issue appears to be that images simply aren't copied to the `publish` folder. They are copied to the (for example) `net7.0-windows10.0.19041.0\win10-x64` folder, which is the parent of the `publish` folder, however. And if you manually copy those same images (having the appropriate "scale-100" filenames) into `publish`, then the published app loads the images correctly.

**Comment by nor0x** [comment]
this is still an issue on .NET 7 😴

**Comment by the1killer** [comment]
Seeing this as well with just **`dotnet publish -f net7.0-windows10.0.19041.0 -c Release -p:WindowsPackageType=None`**

**Comment by lothrop** [comment]
Yes, still happening in .NET 7. My workaround is to skip publishing and just use the bin folder.

**Comment by Captnwalker1** [comment]
My workaround, a powershell script that runs via a afterPublish target:
```
#fixMauiPublishImages.ps1
param
(
	[Parameter(Mandatory=$true)][string]$StagingDir,	
	[Parameter(Mandatory=$true)][string]$OutputDir
)

#if maui ever fixes this issue, remove this script https://github.com/dotnet/maui/issues/10526

$OutputPath = "$OutputDir\Your.Project.Name"

#Copy only the files we want, but preserving directory structure
Write-Host "Copying file(s)..."
robocopy $StagingDir $OutputDir *.png *.ico *.html *.svg *.sql *.ttf *.otf /s /nfl /ndl

Write-Host "Done."
```

After Publish Target in csproj:
```
...
	<Target Name="CustomAfterPublish" AfterTargets="Publish;AfterPublish">
		<Exec Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'" Command="powershell.exe -ExecutionPolicy Bypass -NoProfile -NonInteractive -File &quot;$(ProjectDir)..\FixMauiPublishImages.ps1&quot; $(OutputPath) $(PublishDir)" />
	</Target>
...
```

**Comment by taylormoyle** [comment]
I am also having issues with images and fonts rendering while remote debugging, and when clicking the .exe file inside the bin folder.

When I run the app on a local Windows emulator it renders the images just fine, but when I either remote debug, click the .exe file in the bin folder, or msbuild publish them, the images and fonts do not render. 

Any advice on how to remedy this would be appreciated.

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.8.0 Preview 5.0. Can repro on windows platform.

**Comment by ejwipp** [comment]
This is super frustrating.  I'm at the end of a Xamarin to MAUI port and thought I was through all of the challenges only to find my published Windows version doesn't display images.  Discovering that this was initially reported in October of 2022 and it is still open is mind blowing.
